docs/index.rst

```diff
@@ -0,0 +1,18 @@
+
+.. centered:: |shieldRepo|_ |shieldGitter|_ |shieldTwitter|_
+
+.. |shieldRepo| image:: https://img.shields.io/badge/VUnit/vunit-0c479d.svg?longCache=true&style=flat-square&logo=github
```

Comment:
The shield points to VUnit/tdd-intro, but shows VUnit/vunit. I suggest having two shields, one for each repo.